### ISSUE TITLE
Finds only every 2nd citation key

### ISSUE THREAD

**Issue by Zocker1999NET** [MalteHei/bibtex-citer#1]
Hey, first of all, thanks for this extension. Assuming it will work completely in the future, this will a big improvement for me while writing my thesis with VSC.

I sadly have the issue that your extension only finds every 2nd citation key in my BibLaTeX file exported by Zotero. Reordering the entries does change which will be recognized, so I believe it has nothing to do with some individual entries.

I further tried to simplify the entries and stopped at following stage, where I still have the same issue (first entry found, second not):

```bibtex
@online{testthat,
	title = {Jump},
	url = {https://example.org},
}

@online{testthis,
	title = {ToThis},
	url = {https://example.com},
}
```

I also tried removing my custom cite commands regexes, which didn't help either.
Can you maybe help me solve this?

**Comment by Zocker1999NET** [comment]
Oh, I debugged the issue with regex101:

The regex engine of javascript is, by default, greedy. I don't understand exactly why, but as the title of the first one it finds the title of the second one, hence skipping the second key.

It can be fixed by declaring the matching group for skipping non-title lines as non-greedy with a question mark: `(…)*?`.

See fixed regex on [regex101](https://regex101.com/r/6x68GN/2), remove the added question mark to see what seems to happen in my case.

I will try to build the extension myself with the fix, and if it works, I will create a PR with it.

**Comment by MalteHei** [comment]
Thanks for your contribution!

After some hassle, I figured out how to publish a new version of this extension.
The changes are now available as `v1.4.0`.
